src/late_multicellular_stage/MulticellularConvolutionDispayer.cs

```diff
@@ -75,17 +76,30 @@ public void DisplayFromLayout(IReadOnlyCollection<MulticellularMetaball> layout)
         Mesh = meshGen.DualContour();
         Mesh.SurfaceSetMaterial(0, material);
 
+        Task uvUnwrap = new Task(() => UVUnwrapAndTexturize((Mesh as ArrayMesh)!));
+        TaskExecutor.Instance.AddTask(uvUnwrap);
+
         CustomAabb = new Aabb(minExtends, maxExtends);
     }
 
-    protected override void Dispose(bool disposing)
+    private void UVUnwrapAndTexturize(ArrayMesh mesh)
     {
-        if (disposing)
-        {
-            material?.Dispose();
-        }
+        var nativeVariant = Variant.From(mesh).CopyNativeVariant();
```

Comment:
What's the logic with CopyNativeVariant? I've never used that before. Why not just `new Variant(mesh)`?

Reply:
I assumed that C++ side wouldn't be able to use a C# Variant, but it actually seems to work

Reply:
It does make sense that the C# bindings would try to match the memory layout and size of the C++ side. Though for safety it might be nice to add a static assert in the C++ code and a startup check in C# side to make sure the size of the types match a known value.